backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/lsm/LiveMigrateDiskCommand.java

```diff
@@ -214,9 +220,29 @@ protected CreateSnapshotForVmParameters getCreateSnapshotParameters() {
         params.setDiskIdsToIgnoreInChecks(getImageGroupIds());
         params.setNeedsLocking(false);
         params.setEndProcedure(EndProcedure.COMMAND_MANAGED);
+
+        updateImageInitialSizeInBytes(params);
```

Comment:
Why not return the disk image map and set it here?

    params.setDiskImagesMap(createDiskImageMap())

The function will return a map with one entry per disk that needs initial size.

Reply:
Done.
Changed to `createDiskImageMap()` that returns a `Map` with either one entry or an empty `Collections.emptyMap()`.